library/src/main/java/com/chuckerteam/chucker/api/internal/data/entity/HttpTransaction.kt

```diff
@@ -118,13 +119,16 @@ internal class HttpTransaction(
             return when (status) {
                 Status.Failed -> " ! ! !  $method $path"
                 Status.Requested -> " . . .  $method $path"
-                else -> responseCode.toString() + " " + method + " " + path
+                else -> responseCode.toString() + " " + method + " " + path + if (isGraphql) operationName else ""
```

Comment:
Can you add a space between path and operation name?

Reply:
done!